app/Models/FeeStatement.php

```diff
@@ -2,7 +2,6 @@
 
 namespace App\Models;
 
-use App\Models\FeeStatement;
```

Comment:
You cant do this. Same logic as the Util.php. You cant use a class that is the same as itself